test/functional/feature_lmdb_backend_test.py

```diff
@@ -40,10 +40,10 @@ def run_test(self):
         assert_equal(self.block_height(), 0)
 
```

Comment:
So I'm writing this comment again here so that it's not lost.

Please try to write functional tests for PoS staking. There are primitives that can be generated (VRF keys, private keys, etc), for which some RPC functions have to be created. In case you're stuck, let's talk about this.